### ISSUE TITLE
Minishift version output contains extra new line

### ISSUE THREAD

**Issue by agajdosi** [minishift/minishift#1204]
Problem is in `minishift/cmd/minishift/cmd/version.go - line 38`.

Actual behaviour:
```
[agajdosi@dhcp-24-156 minishift]$ minishift version
minishift v1.3.1+f4900b07

[agajdosi@dhcp-24-156 minishift]$
```

Expected behaviour:
```
[agajdosi@dhcp-24-156 minishift]$ minishift version
minishift v1.3.1+f4900b07
[agajdosi@dhcp-24-156 minishift]$
```

Line 38 is now:
```
fmt.Println(fmt.Sprintf("minishift v%s+%s\n", version.GetMinishiftVersion(), version.GetCommitSha()))
```
should be:
```
fmt.Println(fmt.Sprintf("minishift v%s+%s", version.GetMinishiftVersion(), version.GetCommitSha()))
```
or maybe just
```
fmt.Printf("minishift v%s+%s\n", version.GetMinishiftVersion(), version.GetCommitSha())
```

**Comment by agajdosi** [comment]
Will fix this tomorrow if it is not taken until then - blocks #1192.

**Comment by hferentschik** [comment]
Any particular reason you want to remove the newline? Why do you think is should not be there. I'd be careful changing anything here w/o double checking with the JBIDE integration.

**Comment by hferentschik** [comment]
> blocks #1192.

I am intrigued, how can this block a test? Assuming for a moment that the new line is indeed intentional, are you saying you could not write a test which verifies this format? If the only reason for this change is driven by a test, I think it is not worth doing.

**Comment by agajdosi** [comment]
@hferentschik

> Any particular reason you want to remove the newline?

It is not consistent, any other command in Minishift is not using extra new line.

> are you saying you could not write a test which verifies this format?

I have no problem with writing it, it is already done. I just thought that it would be better to fix it as I saw it just as a bug and then create PR for the test, so we do not have to change the test day after it is originally pulled.

If the only reason for it is driven by a test on JBIDE side I am ok with closing this as it is actually a feature. I can then send the PR for test of version which will count with current format.

**Comment by hferentschik** [comment]
Merged via pull request #1207
